src/App.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/extensions */
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Navbar from './components/Navbar';
+import HomeScreen from './screens/HomeScreen';
+import LoginScreen from './screens/LoginScreen';
+import RegisterScreen from './screens/RegisterScreen';
+import AddCar from './components/AddCar';
+import UpdateCar from './components/UpdateCar';
+import CarsScreen from './screens/CarsScreen';
+import DetailsCarScreen from './screens/DetailsCarScreen';
+import BookingScreen from './screens/BookingScreen';
+import ReservationsScreen from './screens/ReservationsScreen';
+import { getCars } from './redux/actions/CarAction';
+import UsersScreen from './screens/UsersScreen';
+
+function App() {
```

Comment:
- [ ] Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.


Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/224173413-c8c80b58-a812-49bb-a352-cfab0b37b077.png)